src/webui/www/private/views/preferences.html

```diff
@@ -926,6 +926,14 @@
                     <input type="checkbox" id="coalesceReadsAndWrites" />
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <label for="pieceExtentAffinity">QBT_TR(Use piece extent affinity (requires libtorrent >=1.2.2):)QBT_TR[CONTEXT=OptionsDialog]&nbsp;<a href="https://libtorrent.org/single-page-ref.html#piece_extent_affinity" target="_blank">(?)</a></label>
```

Comment:
I suppose you missed an space between >= and 1.2.2? I would add that anyway.

Reply:
Sure, will fix